bin/ext4mag.py

```diff
@@ -0,0 +1,145 @@
+# ext4mag.py
```

Comment:
can you make sure this conform to pep-8?
https://peps.python.org/pep-0008/

Reply:
fixed